app/components/bookings/edit/edit-label.component.ts

```diff
@@ -0,0 +1,56 @@
+import {Component, Input, Output} from '@angular/core';
+import {IONIC_DIRECTIVES, NavController, Toast} from "ionic-angular/index";
+import {Booking} from "../../../models/booking/booking";
+import {HoursToWorkdaysPipe} from "../../../pipes/hours-to-workdays.pipe";
+import {BookingService} from "../../../services/bookings/booking.service";
+import {EventEmitter} from "@angular/common/src/facade/async";
+
+@Component({
+  selector: 'edit-label',
+  directives: [IONIC_DIRECTIVES],
+  templateUrl: 'build/components/bookings/edit/edit-label.component.html',
+  pipes: [HoursToWorkdaysPipe]
+})
+
+export class EditLabelComponent {
+
+  private _isEditMode: boolean = false;
+  private _rememberValue: String= "";
+
+  @Input()
+  public text: String;
+  @Output()
+  public updateBooking = new EventEmitter<string>();
+
+  constructor() {
+  }
+
+  public focus():void{
+    // this Method is just here for the case that this Editlabel should be triggered
+    // by a click instead of a longclick in the future
```

Comment:
if you'd need it in the future, you could still add it!
There are a lot of event methods, if we would add method stubs for every event, which could be potentially added in the future, we would have a lot of empty methods ;)